TestHelpers.Tests/MockFileCopyTests.cs

```diff
@@ -106,8 +106,19 @@ public void MockFile_Copy_ShouldThrowNotSupportedExceptionWhenSourceFileNameCont
 
             var destFilePath = XFS.Path(@"c:\something\demo.txt");
             var fileSystem = new MockFileSystem();
+            var excludeChars = new []
```

Comment:
This list, comments and all, is introduced four times throughout the PR. I think there's value in pulling this out into a static constant helper file that we can just pull from.

Reply:
Added a class called `Shared` to contain the building of the chars. There isn't a very good name for them though, as they're just chars that don't have `ArgumentException` to be raised. So it's called `SpecialInvalidPathChars`.